lib/src/utf8.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library utf8;
```

Comment:
There is no need to have a library name, and you risk conflict if any other library in the same program also has the name "utf8" (which isn't that unlikely).

Reply:
Done.

Reply:
Done.